package/src/datepicker.js

```diff
@@ -0,0 +1,39 @@
+import flatpickr from "flatpickr"
+
+export default function Datepicker(scope = document) {
+  if (scope === "") {
+    scope = document
+  } else if (scope instanceof String) {
+    scope = document.querySelectorAll(scope)
+  }
+
+  const datepickerInputs = scope.querySelectorAll("input[data-datepicker-type]")
```

Comment:
I know you haven't changed the behaviour, but I am wondering if this could potentially be related to issue https://github.com/AlchemyCMS/alchemy_cms/issues/2196

We are instantiating flatpickr on the **hidden** input fields (as they have the `data-datepicker-type`). That is different compared to the examples shown on the flatpickr website https://flatpickr.js.org/examples/.

I would be surprised, but since I don't see anything obvious it might be worth checking.

Reply:
We are using `input type="text"` as form fields:

1. https://github.com/AlchemyCMS/alchemy_cms/blob/main/lib/alchemy/forms/builder.rb#L32
2. https://github.com/AlchemyCMS/alchemy_cms/blob/main/app/helpers/alchemy/admin/base_helper.rb#L327

Flatpickr hides the form fields and uses a `readonly` field as the display.